src/content/learn/rendering-lists.md

```diff
@@ -246,52 +246,56 @@ img { width: 100px; height: 100px; border-radius: 50%; }
 
 <Pitfall>
 
-Arrow functions implicitly return the expression right after `=>`, so you didn't need a `return` statement:
+एरो फंक्शंस स्वचालित रूप से `=>` के बाद वाले एक्सप्रेशन को रिटर्न कर देते हैं, इसलिए आपको `return` स्टेटमेंट की आवश्यकता नहीं होती।
 
 ```js
 const listItems = chemists.map(person =>
   <li>...</li> // Implicit return!
 );
 ```
 
-However, **you must write `return` explicitly if your `=>` is followed by a `{` curly brace!**
+हालांकि, **आपको `return` को स्पष्ट रूप से लिखना चाहिए यदि आपके `=>` के बाद `{` कर्ली ब्रेस आ रहा है!**
 
 ```js
 const listItems = chemists.map(person => { // Curly brace
   return <li>...</li>;
 });
 ```
 
-Arrow functions containing `=> {` are said to have a ["block body".](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#function_body) They let you write more than a single line of code, but you *have to* write a `return` statement yourself. If you forget it, nothing gets returned!
+एरो फंक्शंस जिनमें `=> {` होता है, उन्हें ["block body".](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#function_body) कहा जाता है। ये आपको एक से अधिक लाइन का कोड लिखने की अनुमति देते हैं, लेकिन आपको स्वयं `return` स्टेटमेंट लिखना पड़ता है। यदि आप इसे भूल जाते हैं, तो कुछ भी रिटर्न नहीं होता!
 
 </Pitfall>
 
-## Keeping list items in order with `key` {/*keeping-list-items-in-order-with-key*/}
+## सूची आइटम्स को क्रम में रखना `key` {/*keeping-list-items-in-order-with-key*/}
 
-Notice that all the sandboxes above show an error in the console:
+ध्यान दें कि ऊपर दिए गए सभी सैंडबॉक्स कंसोल में एक त्रुटि दिखा रहे हैं:
 
 <ConsoleBlock level="error">
 
 Warning: Each child in a list should have a unique "key" prop.
 
 </ConsoleBlock>
 
-You need to give each array item a `key` -- a string or a number that uniquely identifies it among other items in that array:
+आपको प्रत्येक array आइटम को एक `key` देना होगा -- एक स्ट्रिंग या नंबर जो उसे उस array के अन्य आइटम्स में से विशिष्ट रूप से पहचान सके।
 
 ```js
 <li key={person.id}>...</li>
 ```
 
 <Note>
 
-JSX elements directly inside a `map()` call always need keys!
+`map()` कॉल के अंदर सीधे JSX elements को हमेशा keys की आवश्यकता होती है!
 
 </Note>
 
 Keys tell React which array item each component corresponds to, so that it can match them up later. This becomes important if your array items can move (e.g. due to sorting), get inserted, or get deleted. A well-chosen `key` helps React infer what exactly has happened, and make the correct updates to the DOM tree.
 
 Rather than generating keys on the fly, you should include them in your data:
```

Comment:
You forgot to remove the english version for this. Can you remove the lines 291 - 293